frontend/src/elm/Pages/Lobby.elm

```diff
@@ -54,6 +66,27 @@ update msg model =
             in
             ( { model | session = session }, Cmd.map HeaderEvent cmd )
 
+        NotificationEvent notificationMsg ->
+            case notificationMsg of
+                Notification.OnAcceptFromNotification _ gameID ->
+                    let
+                        ( notification, cmd ) =
+                            Notification.update notificationMsg model.notification
```

Comment:
extract the let out of the case block and combine it with the let from the other case branch